src/main/java/org/cbio/gdcpipeline/processor/ClinicalDataProcessor.java

```diff
@@ -0,0 +1,41 @@
+package org.cbio.gdcpipeline.processor;
+
+import org.cbio.gdcpipeline.model.cbio.CBioClinicalDataModel;
+import org.cbio.gdcpipeline.model.cbio.SampleFileModel;
+import org.springframework.batch.item.ItemProcessor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ClinicalDataProcessor implements ItemProcessor<CBioClinicalDataModel, CBioClinicalDataModel> {
+    @Override
+    public CBioClinicalDataModel process(CBioClinicalDataModel patient) throws Exception {
+
+        //CBioClinicalDataModel modified = modifySampleList(patient);
+
+        //return modified;
+
+        return patient;
```

Comment:
Currently the processor is changing various fields in Patient data to conform to accepted cbio values.